docs/extra.css

```diff
@@ -1,26 +1,29 @@
-@import url('https://fonts.googleapis.com/css?family=Open+Sans:300,400,500|Zilla+Slab');
+@import url('https://fonts.googleapis.com/css?family=Zilla+Slab:300');
 
 body::before {
   background-image: none;
   background-color: white;
 }
 
 .navbar {
-  background-image: none;
-  background-color: black;
+  background-image: linear-gradient(to right, rgb(63, 27, 71), rgb(30, 15, 33));
   border-bottom: none;
 }
 
 .navbar-brand {
   font-family: 'Zilla Slab', serif;
+  font-size: 30px;
+  font-weight: 300;
 }
 
 h1, h2, h3, h4, h5 {
-  font-family: 'Open Sans', sans-serif;
+  font-family: 'Zilla Slab', sans-serif;
```

Comment:
__More Purple™__! This is awesome. Just a couple of small tweaks I'd recommend:

(1) set a family system font in body:
(2) use the common system fontset for the h* tags, while keeping the zilla slab font for just `navbar-brand`. generally I think this improves readability overall and keeps the font specifically for the logo itself.

All together, those changes would look like:

```css
body {
   --main-typeface: font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
   font-family: var(--main-typeface);
}

h1, h2, h3, h4, h5 {
    font-family: var(--main-typeface);
}
```

Reply:
i disagree very strongly with using system fonts. the system fonts look nicer on macs than other platforms, and if we use system fonts, you guys _will not see_ what a big chunk of users see, and you'll have a mistaken impression of how good things look. We should design to one consistent standard, and have a non-system backup webfont for cases where zilla slab is not appropriate

Reply:
We use system fonts for all of the iodide properties, don't we? The point is to use the fonts that work best on all platforms, not just mac. Can you provide more information about where they fail? Happy to reassess this strategy but this is not my understanding.